src/components/Launch/LaunchForm/StructInput.tsx

```diff
@@ -21,6 +21,25 @@ muiTheme.typography.h5 = {
     fontWeight: 400
 };
 
+const formatJson = data => {
+    const keys = Object.keys(data);
+
+    if (keys.includes('title')) {
+        const { title, type, format } = data;
+        data['title'] = `${title} (${format ? format : type})`;
```

Comment:
what about to use `${format ?? type}` instead?

Reply:
That’s a good catch, David. I just forgot to replace it with '??' as I wrote separate .js file to write this function. When I used in .js file, it was throwing an error saying SyntaxError: Unexpected token '?'

Reply:
Will push a fix ASAP.

Reply:
I think the error is thrown coz the js file was not using ES6 or later versions.
For ts, it's okay. :)